docs/docs/environment-variables.md

```diff
@@ -110,6 +110,12 @@ render() {
 - `NODE_ENV`
 - `PUBLIC_DIR`
 
+Gatsby also allows you to specify another enviroment variable when runnng in `development`: 
+
+- `ENABLE_GATSBY_REFRESH_ENDPOINT`
+
+If set to true, this will expose a `/__refresh` webhook that is able to receive `POST` requests to refresh the content you are working with. This can be very useful if you are working with dynamic content while writting your applications, and you want your data to be automatically re-sourced everytime a new change happens.
```

Comment:
```suggestion
If set to true, this will expose a `/__refresh` webhook that is able to receive `POST` requests to refresh the content you are working with. This can be very useful if you are working with dynamic content while writing your applications, and you want your data to be automatically re-sourced every time a new change happens.
```